de.metas.contracts/src/main/java/de/metas/contracts/flatrate/process/C_Flatrate_Term_Change.java

```diff
@@ -33,6 +32,8 @@
 
 	public static final String PARAM_TERMINATION_MEMO = I_C_Flatrate_Term.COLUMNNAME_TerminationMemo;
 	public static final String PARAM_TERMINATION_REASON = I_C_Flatrate_Term.COLUMNNAME_TerminationReason;
+	
```

Comment:
please update your save actions like this:
![image](https://user-images.githubusercontent.com/1264789/32877955-399d021a-caa4-11e7-8d4e-6507f37ceedb.png)

goal: reduce meaningless differences for review etc